### ISSUE TITLE
Dialog: close button is only read out as a button

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#1221]
All dialogs have the same problem. The button at the top right to close is only read out as a button by the screen reader. So you cannot know what this button triggers.

**Comment by luniki** [comment]
Duplicate of #1218
